cupy/linalg/_einsum.py

```diff
@@ -444,29 +445,54 @@ def _tuple_sorted_by_0(zs):
 
 
 def einsum(*operands, **kwargs):
-    """einsum(subscripts, *operands, dtype=False)
+    """einsum(subscripts, *operands, dtype=None, optimize=False)
```

Comment:
How about making `optimize` a keyword-only argument? (maybe `dtype` too, as `numpy.einsum` has `out` followed by it. though it can be another issue)

Reply:
ah, it's in docstring. let me withdraw this comment.

Reply:
Oh yes, this first line is needed to overwrite the function signature in Sphinx. This is a fix that can be backported (see the PR description) because the current docstring isn't accurate.

Reply:
I see, thanks!